.gitignore

```diff
@@ -1,5 +1,6 @@
 bin/
 obj/
+packages/
```

Comment:
This change shouldn't be necessary anymore... package references are cached centrally per machine.

Reply:
Done